### ISSUE TITLE
[Functions] ServiceBus annotation updates

### ISSUE THREAD

**Issue by amamounelsayed** [microsoft/azure-maven-plugins#822]
Updated description, We added Azure/azure-functions-java-library#99
Azure/azure-functions-java-library#96

ServiceBusQueueTrigger and ServiceBusTopicTrigger will have new annotation to support isSessionsEnabled.

The C# attributes is IsSessionsEnabled

**Comment by Flanker32** [comment]
@amamounelsayed Could you please provide a bit more information about this feature request?

**Comment by amamounelsayed** [comment]
@Flanker32 We added https://github.com/Azure/azure-functions-java-library/pull/99 
https://github.com/Azure/azure-functions-java-library/pull/96

ServiceBusQueueTrigger and ServiceBusTopicTrigger will have new annotation to support isSessionsEnabled.

The C# attributes is IsSessionsEnabled

I believe we do not need any updates from the plugin. Please confirm. Thank you!

**Comment by Flanker32** [comment]
@amamounelsayed Maven plugin should work with the changes in function library, but we do need to update the [archetype ](https://github.com/microsoft/azure-maven-archetypes/tree/develop/azure-functions-archetype) and update the function library version, will do this job once function library released

**Comment by amamounelsayed** [comment]
@Flanker32 I released snapshot to https://oss.sonatype.org/content/repositories/snapshots/com/microsoft/azure/functions/azure-functions-java-library/1.3.1-SNAPSHOT/ 

When I tried to test my worker it did not generate the 
"bindings" : [ {
"type" : "serviceBusTrigger",
"direction" : "in",
"name" : "message",
"queueName" : "%SBQueueName%",
"isSessionsEnabled" : true,
"connection" : "AzureWebJobsServiceBus"
} ]

I traced the issue and I believe the plugin azure-functions-maven-plugin:1.3.3 uses azure-functions-java-library-1.3.0. 

Can we have a new snapshot from plugin so I can reference it in the worker release?
Please correct me if I am wrong. Thank you so much for your hard work.

**Comment by Flanker32** [comment]
@amamounelsayed Seems that function plugin still uses its own dependencies instead of project's dependencies. We will update the function library version in next release, before it, you may use the following configuration as a workaround.
```
<plugin>
    <groupId>com.microsoft.azure</groupId>
    <artifactId>azure-functions-maven-plugin</artifactId>
    <version>${azure.functions.maven.plugin.version}</version>
    <dependencies>
        <dependency>
            <groupId>com.microsoft.azure.functions</groupId>
            <artifactId>azure-functions-java-library</artifactId>
            <version>${azure.functions.java.library.version}</version>
            <exclusions>
                <exclusion>
                    <groupId>com.microsoft.azure</groupId>
                    <artifactId>azure-functions-maven-plugin</artifactId>
                </exclusion>
            </exclusions>
        </dependency>
    </dependencies>
</plugin>
```

**Comment by amamounelsayed** [comment]
@Flanker32 It worked sir, thank you!

**Comment by amamounelsayed** [comment]
@Flanker32 We will need to get a snapshot for the change for the plugin. Please let us know if you need anything from our side.
Thank you so much and appreciate your help!

cc @anirudhgarg  @fabiocav

**Comment by Flanker32** [comment]
Close as function 1.3.5 has been released, updated function library to 1.3.1.
